### ISSUE TITLE
Enable serial console on boot

### ISSUE THREAD

**Issue by konasimakopoulos** [eellak/epidose#63]
The RPi serial port is user accessible on the final PCB. If the Ansible script was modified to enable it and thus allow shell access, it would significantly help with debugging.

**Comment by dspinellis** [comment]
@stefanos1316 please make it happen.

**Comment by stefanos1316** [comment]
@konasimakopoulos I find that adding this variable from /boot/config.txt will enable it enable_uart=1.
I guess this is what you need, right?

**Comment by dspinellis** [comment]
@stefanos1316 : Please work on each on a separate branch, so that multiple PRs can be open together.

**Comment by konasimakopoulos** [comment]
@stefanos1316: Yes, that should be all that is needed (but please also  check the cmdline.txt file to make sure that the kernel is configured to output console messages on the serial port).

**Comment by stefanos1316** [comment]
@dspinellis ok, I have submitted a PR on a different branch.

**Comment by stefanos1316** [comment]
@konasimakopoulos according to this [document](https://www.raspberrypi.org/documentation/configuration/uart.md) there are two serial ports for RPi Zero. I guess I should enable/configure the primary (PL011), right?

**Comment by konasimakopoulos** [comment]
Yes, that is correct!

**Comment by stefanos1316** [comment]
@konasimakopoulos according to some guides that I read ([1](https://learn.adafruit.com/raspberry-pi-zero-creation/enable-uart)) by just adding the mentioned line will do the magic. Also, what you need in cmdline.txt is this "console=serial0,115200 console=tty1". I have checked RPi3 and RPi Zero, and both of them has it enabled. Unfortunately, I do not have a port to test it out.  Any idea if I can do it in a different way?

**Comment by konasimakopoulos** [comment]
Just checked on a RPi Zero (SD with 2020-12-02-raspios-buster-armhf-lite). Just appending "enable_uart=1" to /boot/config.txt will enable the serial port after a reboot.

**Comment by stefanos1316** [comment]
Great, thanks @konasimakopoulos .
I have submitted a PR addressing this issue.
